recipe/build.sh

```diff
@@ -62,16 +65,31 @@ mv pyproject{-qt5,}.toml
 echo "[tool.sip.project]
 sip-include-dirs = [\"${PREFIX}/lib/python${PY_VER}/site-packages/PyQt5/bindings\", \"${PREFIX}/share/sip\"]" >> pyproject.toml
 
-sip-build \
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" == "1" ]]; then
+  SIP_COMMAND="$BUILD_PREFIX/bin/python -m sipbuild.tools.build"
+  SITE_PKGS_PATH=$($PREFIX/bin/python -c 'import site;print(site.getsitepackages()[0])')
+  EXTRA_FLAGS="--target-dir $SITE_PKGS_PATH"
+fi
+
+$SIP_COMMAND \
     --no-make \
     --qsci-features-dir ../src/features \
     --qsci-include-dir ../src \
     --qsci-library-dir ../src \
-    --api-dir ${PREFIX}/qsci/api/python
```

Comment:
maybe add `--qt-shared`? `pyqtwebkit` has this... If that doesn't work, I'm afraid I'm out of ideas,